lib/pbench/server/api/resources/users_api.py

```diff
@@ -163,9 +144,10 @@ def __init__(self, config, logger, auth):
                 "pbench-server", "token_expiration_duration"
             )
         except NoOptionError:
-            self.token_expire_duration = None
+            # Defaults to 60 minutes
+            self.token_expire_duration = 60
```

Comment:
You should consider referencing this value symbolically.

Policy definitions should be gathered and placed in a strategic location (e.g., at the top of the referencing class definition, or in a separate module) and not sprinkled throughout the code.

Reply:
On reconsideration (as I mentioned elsewhere), I think it would be better to die if we cannot get this value from the server configuration file.  So, it would suffice to just remove the `try` block and let the exception from `server_config.get()` propagate (so long as the result contains enough information to diagnose what I would suggest is a "configuration error"...).

In this way, it is defined in _one place_ (in the server config file), and it doesn't need to have a symbolic representation in the code (other than the `token_expire_duration` attribute).